### ISSUE TITLE
Neural tangents finite network gives different results compared with JAX

### ISSUE THREAD

**Issue by NiuJ1ao** [google/neural-tangents#155]
Hi, I am new to neural tangents. As far as I know, the stax in neural tangents extends the stax in JAX. So, I expect that the ```init_fn``` and  ```apply_fn``` work the same way as in JAX. However, when I try to change the model in the [mnist_classifier.py example](https://github.com/google/jax/blob/main/examples/mnist_classifier.py) to ```neural_tangents.stax```, I get different results as shown below. 

Do I miss anything important here or misunderstand some concepts in neural tangents?

===================================
Starting training JAX FCN...
Epoch 0 in 1.35 sec
Training set accuracy 0.8800333142280579
Test set accuracy 0.8884999752044678
Epoch 1 in 0.24 sec
Training set accuracy 0.9114833474159241
Test set accuracy 0.9194999933242798
Epoch 2 in 0.26 sec
Training set accuracy 0.9248000383377075
Test set accuracy 0.9276999831199646
Epoch 3 in 0.26 sec
Training set accuracy 0.9338666796684265
Test set accuracy 0.9368000030517578
Epoch 4 in 0.26 sec
Training set accuracy 0.9396666884422302
Test set accuracy 0.9398999810218811
Epoch 5 in 0.27 sec
Training set accuracy 0.9438166618347168
Test set accuracy 0.9440999627113342
Epoch 6 in 0.26 sec
Training set accuracy 0.948283314704895
Test set accuracy 0.9483999609947205
Epoch 7 in 0.27 sec
Training set accuracy 0.9510999917984009
Test set accuracy 0.9501999616622925
Epoch 8 in 0.26 sec
Training set accuracy 0.9531500339508057
Test set accuracy 0.9519999623298645
Epoch 9 in 0.26 sec
Training set accuracy 0.9551833271980286
Test set accuracy 0.9528999924659729

Starting training Neural Tangents FCN...
Epoch 0 in 0.65 sec
Training set accuracy 0.09963333606719971
Test set accuracy 0.10130000114440918
Epoch 1 in 0.25 sec
Training set accuracy 0.1040833368897438
Test set accuracy 0.10639999806880951
Epoch 2 in 0.25 sec
Training set accuracy 0.10893333703279495
Test set accuracy 0.11309999972581863
Epoch 3 in 0.25 sec
Training set accuracy 0.1145000010728836
Test set accuracy 0.11779999732971191
Epoch 4 in 0.27 sec
Training set accuracy 0.12003333121538162
Test set accuracy 0.12399999797344208
Epoch 5 in 0.27 sec
Training set accuracy 0.12481667101383209
Test set accuracy 0.1305999904870987
Epoch 6 in 0.27 sec
Training set accuracy 0.1304166615009308
Test set accuracy 0.13589999079704285
Epoch 7 in 0.25 sec
Training set accuracy 0.13751666247844696
Test set accuracy 0.14059999585151672
Epoch 8 in 0.23 sec
Training set accuracy 0.14315000176429749
Test set accuracy 0.14800000190734863
Epoch 9 in 0.24 sec
Training set accuracy 0.1500166654586792
Test set accuracy 0.15479999780654907

**Comment by romanngg** [comment]
Hi Yuchen, thanks for pointing this out, we should highlight the differences more.

`neural_tangents.stax` layers are different in that they always use Lecun/He initialization, i.e. iid Gaussian weights (divided by the square root of the fan in), while `example_libraries.stax` seems to default to Glorot (divided by mean of fan in and fan out) and truncated Gaussian weights. But most infinite-width research derives results for the former case, which is why we use non-truncated iid Gaussian weights divided by the square root of the fan in.

Another important difference (mentioned in https://neural-tangents.readthedocs.io/en/latest/stax.html, bullet point 2) is whether the scaling weights wrt fan in is done as part of weight variance (`parameterization="standard"`, used in `example_libraries.stax`), or as part of the model (`parameterization="ntk"` - the default in `neural_tangents.stax`, but you can change it to `"standard"`). Similarly to the above, we default to the latter since most infinite width results use the `"ntk"` parameterization. For more details on this, see https://arxiv.org/abs/2001.07301 and Remark 1 on page 3 of https://arxiv.org/pdf/1806.07572.pdf.

Due to differences above, in NT you may need to try other learning rates to get the same results.
